lib/tasks/sample_data/order_cycle_factory.rb

```diff
@@ -0,0 +1,111 @@
+require "tasks/sample_data/logging"
+
+class OrderCycleFactory
+  include Logging
+  # rubocop:disable Metrics/MethodLength
+  def create_samples
+    log "Creating order cycles"
+    create_order_cycle(
+      "Freddy's Farm Shop OC",
+      "Freddy's Farm Shop",
+      ["Freddy's Farm Shop"],
+      ["Freddy's Farm Shop"],
+      receival_instructions: "Dear self, don't forget the keys.",
+      pickup_time: "the weekend",
+      pickup_instructions: "Bring your own shopping bags or boxes."
+    )
+
+    create_order_cycle(
+      "Fredo's Farm Hub OC",
+      "Fredo's Farm Hub",
+      ["Fred's Farm", "Fredo's Farm Hub"],
+      ["Fredo's Farm Hub"],
+      receival_instructions: "Under the shed, please.",
+      pickup_time: "Wednesday 2pm",
+      pickup_instructions: "Boxes for packaging under the roof."
+    )
+
+    create_order_cycle(
+      "Mary's Online Shop OC",
+      "Mary's Online Shop",
+      ["Fred's Farm", "Freddy's Farm Shop", "Fredo's Farm Hub"],
+      ["Mary's Online Shop"],
+      receival_instructions: "Please shut the gate.",
+      pickup_time: "midday"
+    )
+
+    create_order_cycle(
+      "Multi Shop OC",
+      "Mary's Online Shop",
+      ["Fred's Farm", "Freddy's Farm Shop", "Fredo's Farm Hub"],
+      ["Mary's Online Shop", "Maryse's Private Shop"],
+      receival_instructions: "Please shut the gate.",
+      pickup_time: "dusk"
+    )
+  end
+  # rubocop:enable Metrics/MethodLength
+
+  private
+
+  def create_order_cycle(name, coordinator_name, supplier_names, distributor_names, data)
```

Comment:
keyword arguments would be perfect here to make calls to it a bit easier to understand. 5 arguments is already a lot.